packages/m2m-gateway/test/mockUtils.ts

```diff
@@ -39,6 +41,27 @@ export function getMockedApiDelegation({
   };
 }
 
+export function getMockedApiTenant(): WithMetadata<tenantApi.Tenant> {
+  return {
+    data: {
+      id: generateId(),
+      attributes: [],
```

Comment:
You could also mock these somehow, maybe just quickly with zod mock

```suggestion
      attributes: generateMock(z.array(tenantApi.TenantAttribute)),
```